cadence/contracts/LendingAprSnapshot.cdc

```diff
@@ -180,18 +178,18 @@ pub contract LendingAprSnapshot {
                 )         
         }
         // Start tracking a new market
-        let poolPublicCap = getAccount(poolAddr).getCapability<&{LendingInterfaces.PoolPublic}>(LendingConfig.PoolPublicPublicPath)
-        assert(poolPublicCap.check() == true, message:
+        let poolPublicCap = getAccount(poolAddr).capabilities.get<&{LendingInterfaces.PoolPublic}>(LendingConfig.PoolPublicPublicPath)
+        assert(poolPublicCap != nil && poolPublicCap!.check() == true, message:
```

Comment:
`account.capabilities.get` was changed in the latest version of Cadence to not return an optional any more. Make sure you're testing with the latest version of the CLI

Reply:
Thanks, corrected with latest version